src/script/components/Icon.tsx

```diff
@@ -38,6 +38,9 @@ const normalizeIconName = (name: string) =>
 const createSvgComponent = (svg: HTMLElement, displayName: string): React.FC<IconProps> => {
   const SVGComponent: React.FC<IconProps> = oProps => {
     const viewBox = svg.getAttribute('viewBox');
+    if (!viewBox) {
+      console.error('Svg icon must have a viewBox attribute');
```

Comment:
Is this log statement necessary? Can you throw an error to catch it in our code?

Reply:
I talked about this with @AndyLnd 
Actually we get a react error but it's not that obviously demonstrating what's going wrong...